Apps/Common/src/Delegates/HMACHashDelegate.cs

```diff
@@ -136,7 +136,7 @@ public IHash Hash(string? key)
         }
 
         /// <inheritdoc />
-        public bool Compare(string key, IHash compareHash)
+        public bool Compare(string? key, IHash compareHash)
```

Comment:
Is this a real change?

Reply:
Yeah, It was causing a smell since the implementation was not implementing the interface exactly